db/migrate/20190916075739_add_integer_discount_to_sold_items.rb

```diff
@@ -0,0 +1,5 @@
+class AddIntegerDiscountToSoldItems < ActiveRecord::Migration[5.2]
+  def change
+    add_column :sold_items, :discount, :integer, default: 0
```

Comment:
Why was disscount field a json earlier and wouldn't it be better if we used a float?